### DISCUSSION TITLE
Node.JS help with @deepgram/sdk3.0.0-alpha.12, error: cannot find module 'deepmerge'

### DISCUSSION THREAD

**Discussion by SteveK888** [deepgram/community#450] [General help] [answered]
Hi, I'm having some problem integrating the @deepgram/sdk3.0.0-alpha.12 into my node.js.

https://www.npmjs.com/package/@deepgram/sdk/v/3.0.0-alpha.12

Getting Error: Cannot find module 'deepmerge'...

What am I defining wrongly?

Code snippet: not working

```
//+ myFunction.js
const myFunction = async( req, res ) => {
   const { createClient } = require( "@deepgram/sdk" );

   const deepgramApiKey = 'keepingthissecretuseyourownAPIkeyhereplease';
   const deepgram = createClient( deepgramApiKey );
   const audio = fs.createReadStream("Uploads/" + "myfile.mp3" ); 

const { result, error } = await deepgram.listen.prerecorded.transcribeFile(
         audio, { 

         model: "nova", } )
 .then(  ( response ) => {
 //+ ...snip...
 ) };

}
module.exports = myFunction;

```
-----------------
Previously was using @deepgram/sdk2.4.0 , this format works OK:
Code snippet:

```
//+ myFunction.js
const myFunction = async( req, res ) => {
   const { Deepgram } = require ( "@deepgram/sdk" );

   const deepgramApiKey = 'keepingthissecretuseyourownAPIkeyhereplease';
   const deepgram = new Deepgram ( deepgramApiKey );

   const audio = fs.createReadStream("Uploads/" + "myfile.mp3" ); 
   const DGResponse = await deepgram.transcription.preRecorded(
   { 
      model: "nova",
      stream: audio,
       ... snip ...
   });
   ... snip ...
}
module.exports = myFunction;
```

Thanks in advance for all your help!

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by lukeocodes** [comment, marked as answer]
Hi @SteveK888 , yes there was an issue with that cut of the release. Can you run the following to update to the latest beta, where that is resolved.

```shell
npm i @deepgram/sdk@v3.0.0-beta.4
```

**Reply by SteveK888** [reply]
Yup, that works! Thank you for your quick reply!
